packages/cozy-harvest-lib/src/index.js

```diff
@@ -7,8 +7,10 @@ const dictRequire = lang => require(`./locales/${lang}.json`)
 
 export default Harvest
 
-export const AccountForm = (props, context) => (
-  <I18n dictRequire={dictRequire} lang={context.lang}>
-    <TranslatedAccountForm {...props} />
-  </I18n>
-)
+export const AccountForm = (props, context) => {
+  return (
```

Comment:
Why with `return` ?

Reply:
Side effects of our test earlier this day.